ui/common/src/main/res/values/strings.xml

```diff
@@ -750,6 +756,17 @@
   <string name="purchase_bank_transer_secure_body">Secure transfer directly from your bank.</string>
   <string name="purchase_bank_transfer_pending_disclaimer">We\'re waiting for the funds to arrive from your bank.</string>
 
+  <!--New Purchase flow-->
+  <string name="purchase_add_method_disclaimer">By adding a payment method you agree and consent to be supplied with the content upon payment and thereby lose your right of withdrawal.</string>
```

Comment:
this string is not required, we'll keep the existing one 
´´´
By proceeding with the payment, you agree and consent to be supplied with the content upon payment and thereby lose your right of withdrawal.
´´´

Reply:
Done